README.md

```diff
@@ -14,5 +14,22 @@ For example, searching for "coordinates of Brighton" returns the response "50.84
 
 Try to keep the locations in alphabetical order of name.
 
+# Contributing
+
+## Building locally
+
+```
+# clone the repository
+git clone git@github.com:scalaspace/scalaspace.github.io.git
+cd scalaspace.github.io
+# make your changes to the code
+
+# build the javascript file
+sbt fullOptJS
+
+# open browser
+open index.html
```

Comment:
This is system dependent. Maybe we can just leave it at. "Then open 'index.html' in your browser."